osquery/requests/serializers/json.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include "osquery/requests/requests.h"
+
+namespace osquery {
+
+/**
+ * @brief JSON Serializer
+ */
+class JSONSerializer : public Serializer {
```

Comment:
Maybe move the serializers into `osquery/text/`? Does is make sense to move transports somewhere else too? We can name `osquery/requests` to `osquery/networking`? So we'd wind up with:

- `osquery/text/serializers`,
- `osquery/text/serializer.h`
- `osquery/networking/requests.h`
- `osquery/networking/transports`

Then we can consolidate serialization into the `Serializer` API?